workspace.go

```diff
@@ -144,12 +144,13 @@ type Workspace struct {
 	TagNames                   []string              `jsonapi:"attr,tag-names"`
 
 	// Relations
-	AgentPool    *AgentPool          `jsonapi:"relation,agent-pool"`
-	CurrentRun   *Run                `jsonapi:"relation,current-run"`
-	Organization *Organization       `jsonapi:"relation,organization"`
-	SSHKey       *SSHKey             `jsonapi:"relation,ssh-key"`
-	Outputs      []*WorkspaceOutputs `jsonapi:"relation,outputs"`
-	Tags         []*Tag              `jsonapi:"relation,tags"`
+	AgentPool           *AgentPool          `jsonapi:"relation,agent-pool"`
+	CurrentRun          *Run                `jsonapi:"relation,current-run"`
+	CurrentStateVersion *StateVersion       `jsonapi:"relation,current-state-version"`
```

Comment:
Can you add this to `workspace_integration_test.go` ?

```
t.Run("with current-state-version included", func(t *testing.T) {
        _, rCleanup := createAppliedRun(t, client, wTest1)
        t.Cleanup(rCleanup)
        
        wl, err := client.Workspaces.List(ctx, orgTest.Name, WorkspaceListOptions{
                Include: String("current-state-version"),
        })
        
        assert.NoError(t, err)
        
        assert.NotEmpty(t, wl.Items)
        
        foundWTest1 := false
        for _, ws := range wl.Items {
                if ws.ID == wTest1.ID {
                        foundWTest1 = true
                        assert.NotNil(t, wl.Items[0].CurrentStateVersion)
                        assert.NotEmpty(t, wl.Items[0].CurrentStateVersion.DownloadURL)
                }
        }
        
        assert.True(t, foundWTest1)
})
```

Reply:
Should we also have something like this for the `CurrentRun` relation?

Reply:
Ideally!